ntp.go

```diff
@@ -365,23 +360,35 @@ func getTime(host string, opt QueryOptions) (*msg, ntpTime, error) {
 		return nil, 0, errors.New("response OriginTime != query TransmitTime") // spoofed packet?
 	}
 
+	if m.OriginTime > dst { // Go 1.9 has monotonic clock preventing that, but 1.8 has not, so it's not panic()
```

Comment:
Is this comment really necessary?

Reply:
I think so. That's a reminder, that this `error` may be converted to `panic` when 1.8 support is dropped.